client/src/components/Feed/Post.js

```diff
@@ -274,7 +274,8 @@ const Post = ({
       extra={
         <span>
           <SvgIcon src={statusIndicator} className="status-icon" />
-          {post?.author?.location ? post.author.location.country : ""}
+		  {post?.author?.location?.city ? `${post.author.location.city}, ` : ""}
+          {post?.author?.location?.country ? post.author.location.country : ""}
```

Comment:
Really weird formatting, Check your prettier settings, and make sure you have ESLint configuration set to true on the .env file for the client. Please reformat.